index.html

```diff
@@ -52,20 +51,20 @@
                         <span class="filter-option__icon material-icons">
                             flag
                         </span>
-                        <select class="filter-option__select filter-option__select--priority">
-                            <option disabled selected value="NO_PRIORITY">No Priority</option>
-                            <option value="LOW_PRIORITY">Low Priority</option>
-                            <option value="MEDIUM_PRIORITY">Medium Priority</option>
-                            <option value="HIGH_PRIORITY">High Priority</option>
+                        <select id="filter-select-priority" class="filter-option__select filter-option__select--priority">
```

Comment:
What happens if extra priority is added in this ?
Can't we make a function that renders this list